frontends/election-manager/src/app.test.tsx

```diff
@@ -474,17 +474,17 @@ test('printing ballots and printed ballots report', async () => {
   fireEvent.click(getByText('Absentee'));
   fireEvent.click(getByText('Test'));
   fireEvent.click(getByText('Official'));
-  fireEvent.click(getByText('Print 1 Official', { exact: false }));
+  fireEvent.click(getByText('Print 1', { exact: false }));
```

Comment:
@arsalansufi why be less specific here?

Reply:
Yes good q! Because I added bolding to this case (https://github.com/votingworks/vxsuite/pull/2249#discussion_r935867860), the text is now broken up in a way that even `{ exact: false }` can't capture

An alternative that would be even more preferable is:

```ts
getByRole('button', name: { 'Print 1 Official Absentee Ballot' })
```

Unfortunately, for a reason I haven't yet been able to pin down, these kinds of retrievals sometimes fail. When I replaced all cases of `Print 1 Official` in this test with the above, the first worked and subsequent didn't, despite the element existing in the DOM